src/util/accessibility/live.ts

```diff
@@ -16,7 +16,7 @@ function getLiveElement(document: any, lazyCreate = false): HTMLElement | null {
 		element.setAttribute('aria-live', 'polite');
 		element.setAttribute('aria-atomic', 'true');
 
-		element.classList.add('visually-hidden');
+		element.style.display = 'none';
```

Comment:
Hmm, I think there is a reason `.visually-hidden` and the likes of it exist (ex. `.cdk-visually-hidden` in material). 

I don't think you should use `display: none` and `visiblity: hidden` if you want the the content to be read by the screen reader.

You should either inline all the styles necessary, or we should allow providing a `Live` service at a different level.

WDYT?

Reply:
You are 100% right, updated with the inline style taken from Bootstrap [visually-hidden](https://github.com/twbs/bootstrap/blob/8a965cadf80ed05d16dfe68a35f244cc5698ea8f/scss/mixins/_visually-hidden.scss#L8), noticed that the [cdk-visually-hidden class](https://github.com/angular/components/blob/59c8bb9a14c61c3932d6701994366f05ef719a91/src/cdk/a11y/_index.scss#L4C21-L4C21) might be a little more robust, let me know if you'd like to use this one instead.

Regarding inline style VS providing the `Live` service at a different level, was not really into the service providing based solution because imo it seemed a little bit too complex compared to the inline style one. But seeing the result, I'll give a try to see how it goes because maintaining this inline styling and not benefiting from what could be done on Bootstrap side is kind of sad.